### DISCUSSION TITLE
Share your configs!

### DISCUSSION THREAD

**Discussion by andresestrella** [glzr-io/glazewm#409] [Show and tell]
as title suggests, show off and share your configs here. 
Won't share mine bc mine are just the defaults from the project's readme😅

**Comment by MyGitHubBlueberry** [comment]
Here is my config. Stars are appreciated 🌟 
![image](https://github.com/glzr-io/glazewm/assets/105305430/63a11bec-615a-4ac2-bb52-02d307762083)

Upd: I updated color scheme, btw
![image](https://github.com/glzr-io/glazewm/assets/105305430/794a7035-3f70-4bb9-9903-7d0c4da03426)

https://github.com/MyGitHubBlueberry/glazeWM_config

**Comment by zvbt** [comment]
![image](https://github.com/glzr-io/glazewm/assets/61088560/5d853a7d-48df-474c-a8bf-8b68e319cd92)
Really simple config with Gruvbox color scheme.
https://github.com/zvbt/dotfiles/blob/main/glaze-wm/config.yaml

**Reply by MyGitHubBlueberry** [reply]
nice, I like gruvbox

**Reply by MyGitHubBlueberry** [reply]
btw, here is gruvbox-material color scheme. I find it more pleasant to the eyes. So I'll leave it here  in case you wanna check it out.
https://github.com/sainnhe/gruvbox-material

**Comment by warden7383** [comment]
Main monitor:
![image](https://github.com/glzr-io/glazewm/assets/119640854/c0cc4144-a608-4031-9c5e-daa0ac034ad5)

Secondary monitor:
![image](https://github.com/glzr-io/glazewm/assets/119640854/d14b5253-66cc-4dbd-872b-ae73c83281b9)

Repo:
https://github.com/warden7383/glazeConfig/blob/master/config.yaml

**Reply by MyGitHubBlueberry** [reply]
cool

**Comment by akkiirah** [comment]
![330944471-e04e0688-5a81-43cd-b63c-911a0b96ac3a](https://github.com/glzr-io/glazewm/assets/46369555/0590f4be-85a0-472d-89a3-f04d11d192bc)
Here's my config: https://github.com/akkiirah/glazeConfig

**Comment by aaronedev** [comment]
![2024-05-19 22_48_27-Greenshot](https://github.com/glzr-io/glazewm/assets/44505317/18b3a2c5-ff22-4cb9-b7d7-da2d6cffa6e6)
![2024-05-19 22_47_01-Greenshot](https://github.com/glzr-io/glazewm/assets/44505317/786a9adf-0871-471e-b112-5e1d0ffcb21b)

**glazewm config:**
- https://github.com/aaronedev/glazewm

**wallpaper64 config:**
- https://steamcommunity.com/sharedfiles/filedetails/?id=3188957645
